VideoLocker/src/main/java/org/edx/mobile/view/CourseDiscussionResponsesFragment.java

```diff
@@ -81,9 +81,8 @@ public void onViewCreated(View view, Bundle savedInstanceState) {
                 discussionThread.getIdentifier(),
                 discussionThread.getType() == DiscussionThread.ThreadType.QUESTION);
 
-        // Using application context to prevent activity leak since adapter is retained across config changes
         courseDiscussionResponsesAdapter = new CourseDiscussionResponsesAdapter(
-                getActivity().getApplicationContext(), this, discussionThread);
+                getActivity(), this, discussionThread);
```

Comment:
Yeah, looking over the adapter initializations, they're all being performed in `onViewCreated()`, and since we're also injecting the views as local variables at the moment, there doesn't seem to be any point in being careful with the Context that's being provided to the adapter.